streams/delimiter_stream.ts

```diff
@@ -155,10 +159,10 @@ export class DelimiterStream extends TransformStream<Uint8Array, Uint8Array> {
             const chunkSliced = chunk.subarray(chunkStart, delimitedChunkEnd);
             const result = concat(...bufs, chunkSliced);
             bufs.length = 0;
+            controller.enqueue(result);
```

Comment:
**comment:** This reordering is done so that all branches would agree on the order of actions: First enqueue, then move chunk start pointer.